### PR TITLE
Marker links

### PR DESCRIPTION
**Pull request by jiri-fiala**
* New article "Markers"
* added links to documented markers
* basic info about creating a marker
* closes issue #237 

can be previewed [here](https://devcenter-jfiala.rhcloud.com/en/managing-markers.html)

### CODE REVIEW

File: lib/en/managing-markers.adoc [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+== Using markers
+Enabling a marker is as simple as creating an empty file of a given name in the `.openshift/markers/` directory and link:managing-modifying-applications.html[pushing] it to your repository, in order to trigger the required functionality.
+
+TIP: For example, creating the `.openshift/markers/hot_deploy` in an PHP app will prevent the Apache process from being restarted during the link:managing-deployments.html[deployment].
+
+Creating an empty file in the current directory is typically done using `touch marker_file` on Linux and Mac OS systems or by `copy NUL > marker_file` when on Windows.
+
+[[marker-files-overview]]
+== Marker files overview
+
```
Comment (jsvgoncalves) [inline lib/en/managing-markers.adoc]: Some markers are general, such has `hot_deploy` and `force_clean_build`. How about adding a table here with the generic markers?

File: lib/en/managing-markers.adoc [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+nav_priority: 82
+meta_desc: Control your app using markers.
+---
+= Markers
+
+[[top]]
+[float]
+= Markers
+[.lead]
+Marker files are used to set conditions for various stages of a cartridge's life cycle, enabling you to control certain aspects of the cartridge.
```
Comment (jsvgoncalves) [inline lib/en/managing-markers.adoc]: I would reword this, maybe use the version from the [nodejs markers](https://devcenter-jfiala.rhcloud.com/en/node-js-project-structure.html#markers) page

> Marker files can be used to customize various aspects of the build and deployment process.

**Comment by jsvgoncalves** [comment]
@corey112358 Is this what you expected with #237 ?

**Comment by corey112358** [comment]
@jsvgoncalves how about we link to the other more specific pages about markers for each language also

**Comment by jsvgoncalves** [comment]
@corey112358 Something else than this? 
https://devcenter-jfiala.rhcloud.com/en/managing-markers.html#cartridge-specific-markers
